pkg/service/service.go

```diff
@@ -251,12 +251,25 @@ func (s *DeviceService) selfRegister() errors.EdgeX {
 	return nil
 }
 
+// DriverConfigs retrieves the driver specific configuration
+func (s *DeviceService) DriverConfigs() map[string]string {
+	return s.config.Driver
+}
+
+func (s *DeviceService) SetDeviceOpState(name string, state models.OperatingState) error {
+	// workaround: the device-service-sdk's uses core-contracts for the API URLs,
+	// but the metadata service API for opstate updates changed between v1.1.0 and v1.2.0.
+	d, err := s.GetDeviceByName(name)
+	if err != nil {
+		s.LoggingClient.Error("no such device")
+		return err
+	}
+
+	d.OperatingState = state
+	return s.UpdateDevice(d)
+}
+
 // RunningService returns the Service instance which is running
 func RunningService() *DeviceService {
 	return ds
 }
-
-// DriverConfigs retrieves the driver specific configuration
```

Comment:
Can't remove this yet since it would be a breaking change. Please add TODO to remove it in EdgeX 3.0

Reply:
I have added the function back and added the todo